ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-stable-4.17-upgrade-from-stable-4.17.yaml

```diff
@@ -0,0 +1,330 @@
+      EXTRACT_MANIFEST_INCLUDED: "true"
+      FIPS_ENABLED: "true"
+    test:
+    - chain: openshift-upgrade-qe-sanity-disconnected
+    workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private-cco-manual-security-token-service-private-s3-with-ep-sts-ec2-elb
+- as: aws-ipi-disc-priv-tp-f28
+  cron: 35 3 25 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      ENABLE_OTA_TEST: OCP-69968
+      EXTRACT_MANIFEST_INCLUDED: "true"
+      FEATURE_SET: TechPreviewNoUpgrade
+    test:
+    - chain: openshift-upgrade-qe-sanity-disconnected
+    workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private
+- as: aws-ipi-proxy-sts-f28
+  cron: 26 19 10,26 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      EXTRACT_MANIFEST_INCLUDED: "true"
+    test:
+    - chain: openshift-upgrade-qe-sanity
+    workflow: cucushift-installer-rehearse-aws-ipi-proxy-cco-manual-security-token-service
+- as: aws-ipi-shared-vpc-phz-sts-fips-f28
+  cron: 14 7 6,20 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      EXTRACT_MANIFEST_INCLUDED: "true"
+      FIPS_ENABLED: "true"
+    test:
+    - chain: openshift-upgrade-qe-sanity
+    workflow: cucushift-installer-rehearse-aws-ipi-shared-vpc-phz-sts
+- as: aws-ipi-usertags-custom-sg-fips-f28
+  cron: 55 4 4,20 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      FIPS_ENABLED: "true"
+    test:
+    - chain: openshift-upgrade-qe-sanity
+    workflow: cucushift-installer-rehearse-aws-ipi-usertags-custom-security-groups
+- as: aws-usgov-ipi-private-sts-f28
+  cron: 47 23 7,21 * *
+  steps:
+    cluster_profile: aws-usgov-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      EXTRACT_MANIFEST_INCLUDED: "true"
+    test:
+    - chain: openshift-upgrade-qe-sanity
+    workflow: cucushift-installer-rehearse-aws-usgov-ipi-private-cco-manual-security-token-service
+- as: azure-ipi-fullyprivate-proxy-f28
+  cron: 23 7 16,30 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+    test:
+    - chain: openshift-upgrade-qe-sanity
+    workflow: cucushift-installer-rehearse-azure-ipi-fullyprivate-proxy
+- as: azure-ipi-usertags-f28
+  cron: 36 20 6,20 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+    test:
+    - chain: openshift-upgrade-qe-sanity
+    workflow: cucushift-installer-rehearse-azure-ipi-usertags
+- as: azure-mag-ipi-fips-f28
+  cron: 43 9 3,19 * *
+  steps:
+    cluster_profile: azuremag-qe
+    env:
+      BASE_DOMAIN: qemag.azure.devcluster.openshift.com
+      FIPS_ENABLED: "true"
+    test:
+    - chain: openshift-upgrade-qe-sanity
+    workflow: cucushift-installer-rehearse-azure-mag-ipi
+- as: azure-mag-ipi-fullyprivate-f28
```

Comment:
To align with stable e2e ci jobs, we can remove this job now, use `azure-mag-ipi-fips` to cover azure-mag

Reply:
Same with above comment, need confirm with others:
1. copy from 4.16
2. or only align with 4.17 e2e

Reply:
They are not conflicting. The jobs we expect should be either
copying from 4.16 z stream to 4.17 and updating/adding jobs for those changed in 4.17
or
adding jobs for 4.17 e2e

You can decide which way to use based on your preference.

Reply:
To easily add new stable jobs, I want to only choose e2e jobs, do you think it will meet our test strategy?
for example, I only add 4.17 stable -> stable jobs which exists in 4.17 stable e2e config file. I will not copy 4.16 stable -> stable jobs.

Reply:
> I will not copy 4.16 stable -> stable jobs.

Some existing ci jobs are added on purpose to cover OCP cases, e.g: aws-ipi-disc-priv-tp-f28 and gcp-ipi-proxy-tp-f28, expect them, I am okay with only adding 4.17 stable -> stable jobs which exists in 4.17 stable e2e config file.

Reply:
right now all stable->stable jobs are coming from related e2e jobs.

Reply:
> Some existing ci jobs are added on purpose to cover OCP cases, e.g: aws-ipi-disc-priv-tp-f28 and gcp-ipi-proxy-tp-f28,

You missed the cases.

Reply:
Where are they? I can not see them in https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-stable.yaml

Reply:
Though we are agree to use 4.17 e2e stable job to run stable -> stable z-stream testing, there is an exception. Some stable upgrade jobs are added on purpose in 4.16 stable upgrade ci jobs. E.g: aws-ipi-disc-priv-tp-f28 and gcp-ipi-proxy-tp-f28 in openshift-openshift-tests-private-release-4.16__amd64-stable-4.16-upgrade-from-stable-4.16.yaml , we have to copy them to to 4.17 stable upgrade ci, unless we confirmed with case author, we does not need them any more.

@jiajliu I saw `OCP-69968` and `OCP-69948` are your assigned cases, can you help confirm if we need to copy them from 4.16 stable z-stream to 4.17 stable z-stream upgrade ci.

Reply:
done